toqito/rand/random_density_matrix.py

```diff
@@ -31,7 +31,7 @@ def random_density_matrix(
 
     >>> from toqito.rand import random_density_matrix
     >>> complex_dm = random_density_matrix(2)
-    >>> complex_dm
+    >>> complex_dm # doctest: +SKIP
```

Comment:
Does the need for this warning go away if we specify the type of the output?

Reply:
Not really. 

The output is supposed to be randomly generated density matrix. The expected value can differ over each run of `make doctest`. So, an easier option was to skip this test.